### PR TITLE
Fix package version

### PR DESCRIPTION
**Pull request by jbms**
This fixes the version number issue https://github.com/jbms/sphinx-immaterial/pull/64#issuecomment-1107765536.

### CODE REVIEW

**Comment by 2bndy5** [comment]
I think those pictures were my mistake from when I first revised the docs...

File: .gitattributes [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+*.md text eol=lf
+*.toml text eol=lf
+*.txt text eol=lf
+*.scss text eol=lf
+*.ts text eol=lf
+*.json text eol=lf
+*.html text eol=lf
+*.css text eol=lf
+*.tsx text eol=lf
+*.svg text eol=lf
```
Comment (2bndy5) [inline .gitattributes:16]: *.sh and *.xml from for files added in #64

Comment (jbms) [reply .gitattributes:16]: Done.

**Review by 2bndy5 — APPROVED** [review]

**Comment by 2bndy5** [comment]
I found this cmd helpful when analyzing what files have which line ending
```
git ls-files --eol
```

**Pull request merged**